### ISSUE TITLE
RTL says Not Synced despite the chain being up to date

### ISSUE THREAD

**Issue by kevATin** [raspiblitz/raspiblitz#4381]
**Describe the bug**
Ride The Lightning seems to believe that the chain is not up to date, despite that most certainly not being the case.

**To Reproduce/Steps I took**
After a few months of inactivity I decided to spin up my RaspiBlitz again and after a few days of syncing it worked fine. Even the channel I had opened ages ago was still open.
Then I updated to the latest RaspiBlitz version and everything still looked fine. But I'd like to close the channel so i can move all the funds away. 
I tried to close it via the main menu which just gave me "An error occurred: server is still in the process of starting" same with the Force option. This is shown even after the node is running for hours and at the latest block height.
When I look inside RTL the Chain Status is red and says "Not Synced to Chain", which doesn't make sense to me especially since above it says "Your node is up and running."
The one channel is have is now listed as inactive in RTL but doesn't look like it's getting closed.

**Debug file**

[debug_log.txt] Removed

Any ideas what's the issue here and how to fix it?

**Comment by kevATin** [comment]
Update:
I had tried to reboot it again, trying to make sure everything is running correctly, etc and came back after a few hours with no success, everything was still the same.

So i left the RPi running and yesterday evening i checked back on it out of curiosity and RTL reported that the Chain was synced now (despite that having definitely been the case before already). So I tried to (not forcefully) close the channel and RTL took the request without errors (I don't think i wrote it in the previous comment, but I did try closing via RTL too before).
Then I left it until today and lo and behold, the channel is closed and the BTC are displayed as on-chain again.

So not sure what exactly the issue was. If my server has been running for hours and a button tells me that something is still in the process of starting up, then i'll assume something's broken. Maybe the error message could be clarified in the interface. 
If the lightning process really was doing work in the background all this time, trying to finish syncing something, then it'd probably be good to display that to the user somewhere in the webinterface (the CLI also gave me no indication that it wasn't finished yet, or at least i couldn't find anything obvious).


TLDR: After two days RTL figured out that the chain was synced and allowed me to close the channel. 
If you find this issue because you have a similar problem, try rebooting and then just leave it running for 2-3 days, then check back- maybe it fixed itself.
